### PR TITLE
Build Multi-Platform Docker Image

### PR DESCRIPTION
**Pull request by yordanovsstoyan**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "@types/testing-library__jest-dom": "^5.14.1",
     "babel-jest": "^27.3.1",
     "canvas": "^2.11.0",
-    "dprint": "^0.30.2",
+    "dprint": "0.39.1",
```
Comment (disrupted) [inline frontend/package.json:39]: why not keep the `^`?

File: Dockerfile [hunk 2]
```diff
@@ -42,4 +44,4 @@ RUN pip install -e .
 
 EXPOSE 8080
 
-CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8080"]
+CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8080"]
```
Comment (disrupted) [inline Dockerfile:47]: please fix EOF (missing newline)

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -120,8 +138,9 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
       - name: Build & Push
-        uses: docker/build-push-action@v3
+        uses: docker/build-push-action@v5
         with:
+          platforms: linux/amd64,linux/arm64
```
Comment (disrupted) [inline .github/workflows/ci.yml:143]: great! 😍

File: Dockerfile [hunk 4]
```diff
@@ -4,6 +4,8 @@ FROM node:16 AS frontend
 WORKDIR /build
 COPY ./frontend/package.json ./frontend/package-lock.json /build/
 ENV NEXT_TELEMETRY_DISABLED=1
+RUN apt-get -y update && \
+    apt-get -y install gcc build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
```
Comment (JakobEdding) [inline Dockerfile:8]: @disrupted do you think we could avoid installing system libs by updating some Python deps to versions for which wheels are available for ARM?

File: backend/mypy.ini [hunk 5]
```diff
@@ -0,0 +1,5 @@
+[mypy]
+python_version = 3.10
+disallow_untyped_defs = True
+ignore_missing_imports = True
+exclude = ['env/']
```
Comment (JakobEdding) [inline backend/mypy.ini:5]: For me, mypy runs through by getting rid of this file, removing the `--config-file` flag in run_mypy.sh and appending `exclude = ['.venv/']` to the mypy section in setup.cfg. I think we should simplify the config this way.

**Review by JakobEdding — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 4 above)
Comment (yordanovsstoyan) [reply Dockerfile:8]: Probably yes

Comment (JakobEdding) [reply Dockerfile:8]: I took a look and it doesn't seem possible atm. @yordanovsstoyan, can you add a comment here linking to this issue? https://github.com/Automattic/node-canvas/issues/1662

File: backend/mypy.ini (hunk 5 above)
Comment (JakobEdding) [reply backend/mypy.ini:5]: It should be fine to remove this file now

File: frontend/package.json (hunk 1 above)
Comment (disrupted) [reply frontend/package.json:39]: unresolving because I don't see this changed

Comment (yordanovsstoyan) [reply frontend/package.json]: Sorry, idk why my change locally was not pushed

**Review by disrupted — APPROVED** [review]

**Pull request merged**